t/legal-bot/error-invalid-license-mixed/fffe100e5a9a3e7f6d1fd97512215283/mixed-more.Dockerfile

```diff
@@ -1,5 +1,5 @@
 #
-# SPDX-License-Identifier: BSD
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This is a test case, not a real license declaration.

Reply:
Aha. Good point. I removed the last commit then.